src/nerve_tools/configure_nerve.py

```diff
@@ -343,8 +343,10 @@ def parse_args(
                         help='weight to advertise each service at. Defaults to # of CPUs')
     parser.add_argument('--labels-dir', type=str, default=DEFAULT_LABEL_DIR,
                         help='Directory containing custom labels for nerve services.')
-    parser.add_argument('--envoy-admin-port', type=int, default=9901,
-                        help='Port for envoy admin to get configured envoy listeners.')
+    parser.add_argument('--envoy-admin-port', type=int, default=0,
+                        help='Port for envoy admin to get configured envoy '
+                        'listeners. Defaults to 0 to indicate envoy is '
+                        'disabled.')
```

Comment:
Probably better to just remove the `default=` part at all.  `admin_port ` will be `None` if `--envoy-admin-port` is not specified.

Reply:
+1 especially since "port 0" often has special meaning

Reply:
Did so and updated tests to match.